src/frontend/screens/Audio/CreateRecording/RecordingDone.tsx

```diff
@@ -0,0 +1,235 @@
+    openOnMount: false,
+  });
+
+  useEffect(() => {
+    navigation.setOptions({
+      headerShown: true,
+      headerLeft: props => (
+        <HeaderBackButton
+          {...props}
+          onPress={() => {
+            const audioRecording = {
+              createdAt: Date.now(),
+              duration,
+              uri,
+            };
+            addAudioRecording(audioRecording);
+            setModalContentType('success');
+            openSheet();
+          }}
+          backImage={props => <CloseIcon color={props.tintColor} />}
+        />
+      ),
+    });
+  }, [navigation, addAudioRecording, duration, uri, openSheet]);
+
+  const handleDelete = () => {
+    closeSheet();
+    setPendingAction('delete');
+    onDelete();
+  };
+
+  const handleDeletePress = () => {
+    setModalContentType('delete');
+    openSheet();
+  };
+
+  const handleReturnToEditor = () => {
+    closeSheet();
+    setPendingAction('returnToEditor');
+  };
+
+  const handleRecordAnother = async () => {
+    closeSheet();
+    setPendingAction('recordAnother');
+  };
+
+  const onModalDismiss = () => {
+    switch (pendingAction) {
+      case 'delete':
+        if (navigation.canGoBack()) {
+          navigation.goBack();
+        } else {
+          navigation.navigate('ObservationCreate');
+        }
+        break;
+      case 'returnToEditor':
+        navigation.navigate('ObservationCreate');
+        break;
+      case 'recordAnother':
+        onRecordAnother();
+        navigation.navigate('Audio');
+        break;
+      default:
+        break;
+    }
+    setPendingAction(null);
+  };
+
+  const renderModalContent = () => {
+    if (modalContentType === 'delete') {
+      return (
+        <BottomSheetModalContent
+          icon={<ErrorIcon />}
+          title={t(m.deleteBottomSheetTitle)}
+          description={t(m.deleteBottomSheetDescription)}
+          buttonConfigs={[
+            {
+              dangerous: true,
+              text: t(m.deleteBottomSheetPrimaryButtonText),
+              icon: <DeleteIcon color={WHITE} />,
+              onPress: handleDelete,
+              variation: 'filled',
+            },
+            {
+              variation: 'outlined',
+              text: t(m.deleteBottomSheetSecondaryButtonText),
+              onPress: closeSheet,
+            },
+          ]}
+        />
+      );
+    } else if (modalContentType === 'success') {
+      const description = (
+        <View style={{marginTop: 40}}>
+          <Text style={{fontSize: 16}}>
+            <FormattedMessage
+              {...m.successDescription}
+              values={{
+                audioRecording: t(m.audioRecording),
+                bold: message => (
+                  <Text style={{fontWeight: 'bold'}}>{message}</Text>
+                ),
+              }}
+            />
+          </Text>
+        </View>
+      );
+      return (
+        <View style={{marginTop: 80}}>
```

Comment:
Also instead of wrapping this, you can just add a `marginTop` of 80 directly to the `SuccessIcon`